vendor/diff-stdlib-format.patch

```diff
@@ -0,0 +1,222 @@
+--- ocaml-4.13-upstream/format.ml
++++ ocamlformat_support/format.ml
+@@@@
+   | Pp_tbegin of tbox          (* beginning of a tabulation box *)
+   | Pp_tend                    (* end of a tabulation box *)
+   | Pp_newline                 (* to force a newline inside a box *)
+   | Pp_if_newline              (* to do something only if this very
+                                   line has been broken *)
++  | Pp_string_if_newline of string
++                               (* print a string only if this very
++                                  line has been broken *)
++  | Pp_or_newline of int * int * string * string
++                               (* print a break and the first string if this
++                                  very line has not been broken, otherwise
++                                  print the second string *)
++  | Pp_fits_or_breaks of int * string * int * int * string
++                               (* print a string if the enclosing box fits,
++                                  otherwise print a break and a string *)
+   | Pp_open_tag of stag         (* opening a tag name *)
+   | Pp_close_tag               (* closing the most recently open tag *)
+ 
+ and stag = ..
+ 
+@@@@
+   format_string state before;
+   pp_output_newline state;
+   state.pp_is_new_line <- true;
+   let indent = state.pp_margin - width + offset in
+   (* Don't indent more than pp_max_indent. *)
+-  let real_indent = Int.min state.pp_max_indent indent in
++  let real_indent = min state.pp_max_indent indent in
```

Comment:
This change does not seem to be needed.

Reply:
That's because min and max are only available since 4.13: https://github.com/ocaml/ocaml/blob/trunk/stdlib/int.mli

Reply:
done, we had a small lib `parser_shims` redefining these for the parser files already.